yapapi/runner/__init__.py

```diff
@@ -84,7 +83,7 @@ def __init__(
         budget: Union[float, Decimal],
         strategy: MarketStrategy = DummyMS(),
         subnet_tag: Optional[str] = None,
-        event_emitter: Optional[Callable[[Event], None]] = None,
+        event_consumer: Optional[Callable[[Event], None]] = None,
```

Comment:
There's also `_emit` field in `runner.task.Task` and `_emitter` in `runner.ctx.WorkContext` and `runner.ctx._RecvFile`. Let's rename them also (to `_event_consumer`?)

Reply:
@azawlocki hmm, after another look at the code I updated the `_wrapped_emitter` as `_wrapped_consumer` in the `Engine` itself ... but for the others that you mentioned, I'm not so sure ... since, actually, they do _emit_ those events that the consumer finally receives ...